docs/standard/garbage-collection/fundamentals.md

```diff
@@ -136,11 +136,11 @@ A garbage collection has the following phases:
 
   Because generation 2 collections can occupy multiple segments, objects that are promoted into generation 2 can be moved into an older segment. Both generation 1 and generation 2 survivors can be moved to a different segment, because they are promoted to generation 2.
 
-  Ordinarily, the large object heap is not compacted, because copying large objects imposes a performance penalty. However, starting with the .NET Framework 4.5.1, you can use the <xref:System.Runtime.GCSettings.LargeObjectHeapCompactionMode%2A?displayProperty=nameWithType> property to compact the large object heap on demand.
+  Ordinarily, the large object heap is not compacted, because copying large objects imposes a performance penalty. However, in .NET Core and in .NET Framework starting with .NET Framework 4.5.1, you can use the <xref:System.Runtime.GCSettings.LargeObjectHeapCompactionMode%2A?displayProperty=nameWithType> property to compact the large object heap on demand.
```

Comment:
>property to compact the large object heap on demand [](start = 306, length = 51)

in core 3.0, LOH is automatically compacted when running with a hardlimit either by specifying a memory limit on a container or the configs. explained in [https://github.com/dotnet/coreclr/pull/22180](https://github.com/dotnet/coreclr/pull/22180)